src/helics/common/logger.h

```diff
@@ -29,6 +29,9 @@ namespace helics
 {
 class LoggingCore;
 
+constexpr int always_log = -100000;  //!< level that will always log
```

Comment:
One of the guidelines that appears some collections of hpc/scientific software library "collections" (RADIUSS at the lab, another one focused on what seemed to be scientific computation math libraries) was providing a way to silence output.

Reply:
I am working with @eberleim on #758.  I want to revisit some of the logging mechanics and assumptions in that PR,  I think this discussion might belong in that PR about what policy we want to allow.  This was mainly about refactoring and cleaning up some clang-tidy warnings.   In general I agree there should be a way to shut off all output, but there is a natural tension between the different layers over user generated log messages.